src/Symfony/Component/HttpFoundation/Cookie.php

```diff
@@ -72,15 +73,19 @@ public static function fromString($cookie, $decode = false)
      * @param int|string|\DateTimeInterface $expire   The time the cookie expires
      * @param string                        $path     The path on the server in which the cookie will be available on
      * @param string|null                   $domain   The domain that the cookie is available to
-     * @param bool                          $secure   Whether the cookie should only be transmitted over a secure HTTPS connection from the client
+     * @param bool|null                     $secure   Whether the cookie should only be transmitted over a secure HTTPS connection from the client or null to set it later using {@see setSecureDefault()}
```

Comment:
If think the most interesting bit to add is the auto-enabling feature (the `setSecureDefault()` is not that interesting if you ask me)